plugins/com.google.cloud.tools.eclipse.appengine.libraries/schema/com.google.cloud.tools.eclipse.appengine.libraries.exsd

```diff
@@ -0,0 +1,194 @@
+            </annotation>
+         </attribute>
+         <attribute name="id" type="string">
+            <annotation>
+               <documentation>
+                  
+               </documentation>
+            </annotation>
+         </attribute>
+         <attribute name="name" type="string">
+            <annotation>
+               <documentation>
+                  
+               </documentation>
+               <appinfo>
+                  <meta.attribute translatable="true"/>
+               </appinfo>
+            </annotation>
+         </attribute>
+      </complexType>
+   </element>
+
+   <element name="library">
+      <complexType>
+         <choice minOccurs="1" maxOccurs="unbounded">
+            <element ref="jarFile"/>
+         </choice>
+         <attribute name="id" type="string">
+            <annotation>
+               <documentation>
+                  
+               </documentation>
+            </annotation>
+         </attribute>
+         <attribute name="name" type="string">
+            <annotation>
+               <documentation>
+                  
+               </documentation>
+            </annotation>
+         </attribute>
+      </complexType>
+   </element>
+
+   <element name="jarFile">
+      <complexType>
+         <sequence>
+            <sequence minOccurs="0" maxOccurs="unbounded">
+               <choice>
+                  <element ref="exclusionFilter"/>
+                  <element ref="inclusionFilter"/>
+               </choice>
+            </sequence>
+            <element ref="mavenCoordinates"/>
+         </sequence>
+         <attribute name="javadocUri" type="string">
+            <annotation>
+               <documentation>
+                  
+               </documentation>
+            </annotation>
+         </attribute>
+         <attribute name="sourceUri" type="string">
+            <annotation>
+               <documentation>
+                  
+               </documentation>
+            </annotation>
+         </attribute>
+         <attribute name="siteUri" type="string">
+            <annotation>
+               <documentation>
+                  
+               </documentation>
+            </annotation>
+         </attribute>
+      </complexType>
+   </element>
+
+   <element name="exclusionFilter">
+      <complexType>
+         <attribute name="pattern" type="string" use="required">
+            <annotation>
+               <documentation>
+                  
+               </documentation>
+            </annotation>
+         </attribute>
+      </complexType>
+   </element>
+
+   <element name="inclusionFilter">
+      <complexType>
+         <attribute name="pattern" type="string" use="required">
+            <annotation>
+               <documentation>
+                  
+               </documentation>
+            </annotation>
+         </attribute>
+      </complexType>
+   </element>
+
+   <element name="mavenCoordinates">
```

Comment:
Seems to be missing `version`, `packaging`, and `classifier`.  I likely won't need to use `packaging` or `classifier`, but it's trivial to include.  And who knows, maybe we'll need to pull down an `eclipse-bundle`.  `version` will default to `LATEST`, `packaging` to `jar`, and empty classifier.